common/components/WalletDecrypt/WalletDecrypt.tsx

```diff
@@ -226,6 +228,10 @@ const WalletDecrypt = withRouter<Props>(
       }
     }
 
+    public componentWillUnmount() {
+      window.clearTimeout(this.walletTimeoutID);
```

Comment:
I am getting this error here and wasn't quite sure about how to handle this 

![image](https://user-images.githubusercontent.com/434238/46824244-3daa7000-cd5e-11e8-802c-92bfa26b878f.png)